crates/nu-std/std/deprecated_dirs.nu

```diff
@@ -26,7 +26,7 @@ print -e $"
 (ansi red)Warning:(ansi reset) The 'std dirs' module will no longer automatically
 be loaded in the next release. To continue using the Shells
 feature, and to remove this warning, please add the following
-to your startup configuration (typically env.nu or config.nu):
+to your startup configuration \(typically env.nu or config.nu):
```

Comment:
Doesn't it need `\(` as well?

Reply:
> Doesn't it need `\)` as well?

Unless there's a corner case I'm missing, I've only never needed to escape the opening.  I guess it never hurts to escape both - But ... maybe *that's* the corner case ;-)

```nu
$"(ansi red)\(Literal)(ansi reset)"
```

Want me to change it or leave it?